packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -509,8 +511,19 @@ impl NeutronMsg {
     /// * **name** is a name of the schedule;
     /// * **period** is a period of schedule execution in blocks;
     /// * **msgs** is the messages that will be executed.
-    pub fn submit_add_schedule(name: String, period: u64, msgs: Vec<MsgExecuteContract>) -> Self {
-        NeutronMsg::AddSchedule { name, period, msgs }
+    /// * **blocker** is a blocker where schedule will be executed.
+    pub fn submit_add_schedule(
+        name: String,
+        period: u64,
+        msgs: Vec<MsgExecuteContract>,
+        blocker: u64,
```

Comment:
Let's change a signature of the method, so it would accept a string representation of the blocker instead of u64.

And do a conversion from string to u64 inside the helper, cause you already have a conversion methods here:
https://github.com/neutron-org/neutron-sdk/pull/161/files#diff-6198a27407233de845943d780924eeb4c0b2e45d06260705b3b33a219b34362bR113